examples/mc_npt_ethane.rs

```diff
@@ -0,0 +1,107 @@
+// Lumol, an extensible molecular simulation engine
+// Copyright (C) 2015-2016 G. Fraux — BSD license
+extern crate lumol;
+extern crate lumol_input as input;
+
+use lumol::Logger;
+use lumol::sys::{UnitCell, System, Trajectory};
+use lumol::units;
+use lumol::sim::Simulation;
+use lumol::energy::{LennardJones, PairInteraction, PairRestriction, NullPotential};
+use lumol::sim::mc::{MonteCarlo, Translate, Rotate, Resize};
+use lumol::out::{EnergyOutput, PropertiesOutput, TrajectoryOutput}; 
+
+use std::path::Path;
+use std::sync::{Once, ONCE_INIT};
+static START: Once = ONCE_INIT;
+
+fn get_system() -> System {
+    let data_dir = Path::new(file!()).parent().unwrap().join("data");
+    let configuration = data_dir.join("ethane.xyz");
+    let mut system = Trajectory::open(configuration)
+                                .and_then(|mut traj| traj.read_guess_bonds())
+                                .unwrap();
+    system.set_cell(UnitCell::cubic(100.0));
+
+    // Add intermolecular interactions
+    // TraPPE parameters for CH3 - note that masses are not correct.
```

Comment:
Ah I see. Here, it would not make a difference, since we don't output the density.